public/js/components/Header/EditHeader.js

```diff
@@ -66,7 +66,9 @@ class EditHeader extends React.Component {
   renderTakeDownButton = (atomPublishState) => {
     if(atomPublishState.id !== 'draft') {
       return (
-        <button type="button" disabled={atomPublishState.id === 'taken-down'} onClick={this.takeDownAtom} className="toolbar__item toolbar__button">Take down</button>
+          <div className="bold-button-padright">
```

Comment:
As we (kind of) use the BEM naming convention, the padding on the right of this should be a "modifier" of the original class. So this one should have the classes `bold-button bold-button--padright` where the second has the padding but the first keeps all the same styles.

You _could_ even take advantage of pseudoselectors instead, so in your css doing `.bold-button:not(:last-child) { padding-right: 5px; }` for example.

Also, could the button not push itself away from the sides using a margin instead of having a containing div? I don't know that this is possible as they're all flex items and a bit funny.